test/libsolidity/semanticTests/functionTypes/comparison_operator_for_external_function_cleans_dirty_bits.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+  function g() external {}
+  function comparison_operators_for_external_function_pointers_with_dirty_bits() external returns (bool) {
+        function() external g_ptr_dirty = this.g;
+        assembly {
+                g_ptr_dirty.address := or(g_ptr_dirty.address, shl(160, sub(0,1)))
```

Comment:
By the way: we should also probably test this for ``.selector`` (with a shift of ``32`` instead of ``160`` in that case).

Reply:
Added in the latest push